java/agents/software-bug-assistant/README.md

```
@@ -0,0 +1,520 @@
+
+INSERT INTO tickets (title, description, assignee, priority, status) VALUES
+('Search Filter "Date Range" Not Applying Correctly', 'The "Date Range" filter on the search results page does not filter records accurately; results outside the specified date range are still displayed.', 'samuel.green@example.com', 'P2 - Medium', 'Resolved');
+
+INSERT INTO tickets (title, description, assignee, priority, status) VALUES
+('Typo in Error Message: "Unathorized Access"', 'The error message displayed when a user attempts an unauthorized action reads "Unathorized Access" instead of "Unauthorized Access."', 'maria.rodriguez@example.com', 'P3 - Low', 'Resolved');
+
+INSERT INTO tickets (title, description, assignee, priority, status) VALUES
+('Intermittent File Upload Failures for Large Files', 'Users are intermittently reporting that file uploads fail without a clear error message or explanation, especially for files exceeding 10MB in size.', 'frank.white@example.com', 'P1 - High', 'Open');
+```
+
+### 7 - Create a trigger to update the `updated_time` field when a record is updated.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)